UnoCheck/Checkups/LinuxNinjaPresenceCheckup.cs

```diff
@@ -0,0 +1,40 @@
+﻿#nullable enable
+
+using DotNetCheck.Models;
+using DotNetCheck.Solutions;
+using System.Threading.Tasks;
+
+namespace DotNetCheck.Checkups
+{
+	public class LinuxNinjaPresenceCheckup : Checkup
+	{
+		public override bool IsPlatformSupported(Platform platform)
+			=> platform == Platform.Linux;
+
+		public override string Id => "linuxninja";
+
+		public override string Title => "Linux Ninja Build";
+
+		public override TargetPlatform GetApplicableTargets(Manifest.Manifest manifest) => TargetPlatform.SkiaGtk;
+
+		public override async Task<DiagnosticResult> Examine(SharedState history)
+		{
+			var ouput = ShellProcessRunner.Run("ninja", "--version").GetOutput().Trim();
+
+			if (ouput.Contains("not found") || ouput.Contains("No such file or directory") || ouput.Contains("ninja: command not found"))
```

Comment:
This may not work if the system is not english. Could you try and use the exit code instead? Should be 0 if successful.